portfolio/src/main/webapp/index.html

```diff
@@ -105,60 +113,62 @@ <h3>LinkedIn: </h3>
-              pictures is not an option, I really like to take photos and then share them with my 
-              friends and family.</p>
-          <p>Architecture, Buildings, Skyscrapers are one of my favorites things to see and photograph, 
-              they really give me satisfaction and peacefulness. I am not an expert in Art and maybe 
-              that why I like to take pictures of it, it is nice to see pictures of art that I have seen 
-              in the past and I did not understand, and then realize the amazing piece that I was looking at.</p>
-          <p>Last but not least I like to take selfies and pictures with family and friends that are important 
-              to me. I think that keeping memories in photograph an then look for them is very fun and is a 
-              nice way to keep memories alive.</p>
-          <h2>Pictures of my favorite places</h2>
+      <article id="gallery">
+        <h2>Image Gallery</h2>
+        <p>Definitely I really like to photograph views that are amazing for me, one great example 
+            of those are the views of cities that I take from high places, for me, is very interesting 
+            to see nature and streets for another point of view. When I am at a new place, no taking 
+            pictures is not an option, I really like to take photos and then share them with my 
+            friends and family.</p>
+        <p>Architecture, Buildings, Skyscrapers are one of my favorites things to see and photograph, 
+            they really give me satisfaction and peacefulness. I am not an expert in Art and maybe 
+            that why I like to take pictures of it, it is nice to see pictures of art that I have seen 
+            in the past and I did not understand, and then realize the amazing piece that I was looking at.</p>
+        <p>Last but not least I like to take selfies and pictures with family and friends that are important 
+            to me. I think that keeping memories in photograph an then look for them is very fun and is a 
+            nice way to keep memories alive.</p>
+        <h3>Pictures of my favorite places</h3>
+        <div id="img-gallery">
           <div id="section">
-              <p>The sea at San Carlos, Sonora, Mexico.</p>
               <img src="images/places/beach1.jpg" alt="Picture of the sea at San Carlos, Sonora, Mexico.">
+              <p>The sea at San Carlos, Sonora, Mexico.</p>
           </div>
           <div id="section">
-              <p>Two of my favortites skyscrapers at Monterrey</p>
               <img src="images/places/building1.jpg" alt="Picture of two of my favorites skyscrapers at Monterrey">
+              <p>Two of my favortites skyscrapers at Monterrey</p>
           </div>
           <div id="section">
-              <p>View of Monterrey from "Cerro de la silla"</p>
               <img src="images/places/view1.jpg" alt="A view of Monterrey from Cerro de la silla">
```

Comment:
What can you see in the view of Monterrey? Is there water, a beach, etc.? Again, these sorts of details are nice to hear if you are a screen reader user.